plugins/backend-trace-forwarder/src/Cardano/BM/Backend/TraceForwarder.lhs

```diff
@@ -67,18 +67,28 @@ least the |Severity|.
 \subsubsection{Plugin definition}
 \begin{code}
 plugin :: forall a s . (IsEffectuator s a, ToJSON a, FromJSON a)
-       => Configuration -> Trace.Trace IO a -> s a -> Text -> IO (Plugin a)
-plugin config _trace _sb tfid = do
+       => Configuration
+       -> Trace.Trace IO a
+       -> s a
+       -> Text
+       -> IO [LogObject a]
+       -> IO (Plugin a)
+plugin config _trace _sb tfid directLogObjects = do
     opts <- getTextOption config tfid
     let minsev = case opts of
                    Nothing -> Debug
                    Just sevtext -> fromMaybe Debug (readMaybe $ unpack sevtext)
     be :: Cardano.BM.Backend.TraceForwarder.TraceForwarder a <- realize config
     dispatcherThr <- spawnDispatcher config (getTF be)
+    -- The callback 'directLogObjects' is used as a source of LogObjects
+    -- that should be sent additionally, only once after the connection is
+    -- re\-established. The application that uses 'lobemo-backend-trace-forwarder'
+    -- plugin provides this callback.
```

Comment:
This should be a part of the function docstring, in the header.

Reply:
Fixed.